README.md

```diff
@@ -316,7 +316,8 @@ in your web browser.
 
 ![welcome-to-phoenix](https://user-images.githubusercontent.com/194400/76152198-ae210200-60b4-11ea-956f-68935daddfe0.png)
 
-😱 *If you are having problems with the server hanging, try [this](###-problems-getting-the-initial-server-running).*
+😱 If you are having problems with the server hanging, try
+ [this](#problems-with-dependencies)
```

Comment:
FYI: "this" is not very descriptive link text. Accessibility checkers such as Lighthouse will flag it as "generic". 
see: https://accessibility.oregonstate.edu/descriptivelinks and https://web.dev/link-text

Reply:
Mentioned this in issue #24, will be updated in upcoming PR.